src/soundio/sounddevice.cpp

```diff
@@ -36,15 +36,15 @@ void SoundDevice::setSampleRate(double sampleRate) {
     m_dSampleRate = sampleRate;
 }
 
-void SoundDevice::setFramesPerBuffer(unsigned int framesPerBuffer) {
+void SoundDevice::setConfigFramesPerBuffer(unsigned int framesPerBuffer) {
     if (framesPerBuffer * 2 > MAX_BUFFER_LEN) {
         // framesPerBuffer * 2 because a frame will generally end up
         // being 2 samples and MAX_BUFFER_LEN is a number of samples
         // this isn't checked elsewhere, so...
         reportFatalErrorAndQuit("framesPerBuffer too big in "
                                 "SoundDevice::setFramesPerBuffer(uint)");
     }
-    m_framesPerBuffer = framesPerBuffer;
+    m_configFramesPerBuffer = framesPerBuffer;
```

Comment:
This and the constructor seem to be the only place where `m_configFramesPerBuffer` is set. Since the framesPerBuffer is unsigned, it's not possible that it becomes negative. But it's of signed integer type.

Reply:
We use SINT for all sample related index and size variables. This type needs to be signed to allow seeking backwards and is compatible to Qt's `int` size and index type.